main.js

```diff
@@ -293,3 +293,23 @@ openBtn.forEach((button) => {
   });
 });
 // }
+
+// Form Validation
+
+const form = document.querySelector('.contact');
+const error = document.querySelector('.throw-error');
+const mail = document.querySelector('.inputEmail');
+
+form.addEventListener('submit', (event) => {
+  event.preventDefault();
+  if (mail.value !== mail.value.toLowerCase()) {
+    error.style.display = 'block';
+    error.textContent = 'Kindly input lower case characters in all fields';
+    error.style.color = 'red';
+    error.style.fontFamily = 'Roboto, sans-serif';
+    error.style.margin = '24px auto';
+    error.style.border = '1px solid gray';
+  } else {
+    form.submit();
+  }
+});
```

Comment:
- [ ] Good job working on the validation functionality, however, when the email characters are all lowercase, the form is not submitting. Kindly fix it.